### PR TITLE
Update Other, TV, and Ceiling Fan stochastic schedules

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Update the stochastic schedule generator to:
* generate TV schedules that follow the ATUS TV schedule fractions (distinct weekday/weekend) and multipliers (and not Other schedule fractions and multipliers).
* generate Other schedules that follow the [new](https://github.com/NREL/OpenStudio-HPXML/pull/1572) Other schedule fractions (still uses non-constant 2010 BAHSP monthly multipliers).
* generate Ceiling Fan schedules that follow the [new](https://github.com/NREL/OpenStudio-HPXML/pull/1572) Ceiling Fan schedule fractions (also update multipliers to not follow Other multipliers, but rather Ceiling Fan multipliers that are a function of `weather`)

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] ~Schematron validator (`EPvalidator.xml`) has been updated~
- [x] Sample files have been added/updated (`openstudio tasks.rb update_hpxmls`)
- [x] Tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests/test*.rb` and/or `workflow/tests/test*.rb`)
- [ ] ~Documentation has been updated~
- [x] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected changes to simulation results of sample files

### CODE REVIEW

File: BuildResidentialScheduleFile/resources/schedules.rb [hunk 1]
```diff
@@ -138,9 +138,12 @@ def create_stochastic_schedules(args:)
       all_simulated_values << Matrix[*simulated_values]
     end
     # shape of all_simulated_values is [2, 35040, 7] i.e. (geometry_num_occupants, period_in_a_year, number_of_states)
-    plugload_weekday_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekdayFractions, 24, 'weekday')
-    plugload_weekend_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekendFractions, 24, 'weekend')
-    plugload_monthly_multiplier = Schedule.validate_values(Constants.PlugLoadsOtherMonthlyMultipliers, 12, 'monthly')
+    plugload_other_weekday_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekdayFractions, 24, 'weekday')
+    plugload_other_weekend_sch = Schedule.validate_values(Constants.PlugLoadsOtherWeekendFractions, 24, 'weekend')
+    plugload_other_monthly_multiplier = Schedule.validate_values(Constants.PlugLoadsOtherMonthlyMultipliers, 12, 'monthly')
```
Comment (joseph-robertson) [inline BuildResidentialScheduleFile/resources/schedules.rb:143]: Should we update these to follow Table C.3(1) of ANSI/RESNET/ICC 301-2022 Addendum C?

File: BuildResidentialScheduleFile/resources/constants.rb [hunk 2]
```diff
@@ -29,6 +29,18 @@ def self.PlugLoadsOtherMonthlyMultipliers
     return '1.248, 1.257, 0.993, 0.989, 0.993, 0.827, 0.821, 0.821, 0.827, 0.99, 0.987, 1.248'
   end
 
+  def self.PlugLoadsTVWeekdayFractions
+    return '0.014, 0.007, 0.004, 0.003, 0.004, 0.006, 0.010, 0.015, 0.020, 0.025, 0.028, 0.031, 0.033, 0.038, 0.042, 0.046, 0.054, 0.062, 0.080, 0.110, 0.132, 0.125, 0.077, 0.034'
+  end
+
+  def self.PlugLoadsTVWeekendFractions
+    return '0.014, 0.007, 0.004, 0.003, 0.004, 0.006, 0.010, 0.015, 0.020, 0.025, 0.028, 0.031, 0.033, 0.038, 0.042, 0.046, 0.054, 0.062, 0.080, 0.110, 0.132, 0.125, 0.077, 0.034'
+  end
```
Comment (joseph-robertson) [inline BuildResidentialScheduleFile/resources/constants.rb:38]: Should we adopt the separate "ATUS TVs weekday" and "ATUS TV weekend" schedule fractions from which these "average" schedule fractions were derived?

File: Changelog.md [hunk 3]
```diff
@@ -35,6 +35,7 @@ __New Features__
 - Miscellaneous Manual J improvements.
 - Adds more error-checking for inappropriate inputs (e.g., HVAC SHR=0 or clothes washer IMEF=0).
 - Allow alternative label energy use (W) input for ceiling fans.
+- Stochastic schedule for TV plug loads now use TV plug load schedules (instead of mirroring stochastic schedule for miscellaneous plug loads).
```
Comment (joseph-robertson) [inline Changelog.md:38]: Is this actually a bugfix?

File: BuildResidentialScheduleFile/resources/schedules.rb (hunk 1 above)
Comment (joseph-robertson) [reply BuildResidentialScheduleFile/resources/schedules.rb:141]: I'll show how much difference we're talking here.

File: BuildResidentialScheduleFile/resources/constants.rb (hunk 2 above)
Comment (joseph-robertson) [reply BuildResidentialScheduleFile/resources/constants.rb:38]: Looks like most other categories have the same schedule fractions between weekday and weekend.

File: Changelog.md (hunk 3 above)
Comment (shorowit) [reply Changelog.md:45]: Seems like an enhancement to me.

File: BuildResidentialScheduleFile/resources/schedules.rb (hunk 1 above)
Comment (shorowit) [reply BuildResidentialScheduleFile/resources/schedules.rb]: The CI only really shows annual results; this makes a huge change to the timeseries results. Are you sure you want to do this?

Before:
image

After:
image

File: BuildResidentialScheduleFile/resources/constants.rb (hunk 2 above)
Comment (shorowit) [reply BuildResidentialScheduleFile/resources/constants.rb]: Seems like a question for @rajeee or @afontani.

File: BuildResidentialScheduleFile/resources/schedules.rb (hunk 1 above)
Comment (joseph-robertson) [reply BuildResidentialScheduleFile/resources/schedules.rb]: Right, for Other and TV we're moving to constant monthly multipliers.

File: BuildResidentialHPXML/measure.xml [hunk 4]
```diff
@@ -7270,6 +7270,2154 @@
...
       <checksum>D07918C5</checksum>
     </file>
+    <file>
+      <filename>extra_files/base-mf.osm</filename>
+      <filetype>osm</filetype>
+      <usage_type>test</usage_type>
+      <checksum>817B1238</checksum>
+    </file>
+    <file>
+      <filename>extra_files/base-mf.xml</filename>
```
Comment (shorowit) [inline BuildResidentialHPXML/measure.xml:7280]: Can you update the tests so they clean up these files when they're finished? We don't want 2000+ new lines in the measure.xml Use the `teardown` method like we do for other measure tests.

Comment (joseph-robertson) [reply BuildResidentialHPXML/measure.xml:7280]: The test file already includes a `teardown` method that (optionally) cleans up these files. I had disabled it during testing. I will remove the folder and update the measure.xml.

**Review by shorowit — APPROVED** [review]
Code changes look reasonable to me.

File: BuildResidentialScheduleFile/resources/constants.rb (hunk 2 above)
Comment (afontani) [reply BuildResidentialScheduleFile/resources/constants.rb]: @shorowit : I could have sworn we had weekday/weekend schedules broken out at one point. Maybe I am getting confused with the stochastic schedule generator. There are significant schedule differences between weekday and weekends for TVs (and other schedules). I would like the schedules broken out by weekday and weekend. To me, it seems relevant to better capture reality, but I understand if we stay consistent with ANSI/RESNET/ICC 301.

**Comment by joseph-robertson** [comment]
https://github.com/NREL/OpenStudio-HPXML/pull/1572/files#diff-60c54474073e3f5a477f95c4bec1d509031f46c05ed506a7ae8d99a87022c102

Checklist:
- [ ] Change stochastic schedule generator Other inputs to schedule fractions and monthly multipliers in the above
- [ ] Change stochastic schedule generator TV inputs to schedule fractions and monthly multipliers in the above
- [ ]

**Comment by joseph-robertson** [comment]
New comparisons after making recent changes.

Before:
image

After:
image
image

**Review by afontani — APPROVED** [review]

**Pull request merged**